XenditExample/XenditExample/Controllers/CreateTokenViewController.swift

```diff
@@ -24,7 +24,7 @@ class CreateTokenViewController: UIViewController {
         super.viewDidLoad()
         
         // Set Publishable Key
-        Xendit.publishableKey = "xnd_public_development_O4iFfuQhgLOsl8M9eeEYGzeWYNH3otV5w3Dh/BFj/mHW+72nCQR/"
+        Xendit.publishableKey = "xnd_public_development_OYiCfOV3hLWnlMI7ebobSTWWNNekoIN5wSDg+Rxj+2HW+7CkBQV1hQ=="
```

Comment:
Reason for changing publishable key in example?

Reply:
ah, forgot to change it back. it's for testing purposes since the code is still in staging. will change it back